docs/Tutorial.md

```diff
@@ -845,9 +845,11 @@ For this tutorial, since there is no public authentication API, we can use a fak
 
 The `authProvider` must expose 5 methods, each returning a `Promise`:
 
-```jsx
+```tsx
```

Comment:
Please revert this one, we want to keep the documentation in JS only for now

Reply:
The top comment in all the code blocks indicates typescript.
```
// in src/authProvider.ts
```
Which portions should I revert? Only this line about the code block type, or are you also including the rest of my changes here?

My concern is only that if this is just JS and not TS, then my change is wholly unnecessary and we should change all the TypeScript files back to JavaScript. Right now the example tutorial is written with `*.ts` files.

Reply:
The tutorial uses the `react-ts` vite template, because it's the best practice even though it doesn't force the user to write TypeScript code. And we want the tutorial to be gentle and inclusive of non-TS developers, so we use only JS.

So all new files should be `*.ts`  or `*.tsx` to follow the template, but there should ideally be no TypeScript-specific code, or it should be explained.

Reply:
I changed the header back to `` ```jsx `` and IntelliJ raised a concern about JS. Seems like it thinks JS does not support typing, so unless I should proceed with committing anyway, we can't have it both ways.
![image](https://user-images.githubusercontent.com/4553730/215701795-d315e251-8fcc-4570-8632-5fff78c0f9a6.png)

Reply:
Without the type hints, pasting the sample into a TypeScript file will not compile. So I'll defer to your direction 😄

Reply:
But it does work with `` ```js `` instead. Curious and baffling to me.